### ISSUE TITLE
Add support to Fysetc Hotkeys 

### ISSUE THREAD

**Issue by llambion72** [Frix-x/klippain#477]
### Describe the feature or hardware support you'd like

Could you add Fystec hotkeys in the future? 
I've installed this addon on my voron 2.4 and it's very useful feature.

**Comment by Surion79** [comment]
it would be helpful to give some links to what you like to be integrated :)

**Comment by llambion72** [comment]
https://www.fysetc.com/products/fysetc-hot-key-board-voron-skirt-button-pcb-voron-skirt-klipper-pre-installed-pcb-board-with-neopixel-led-for-voron-v2-4-trident-switchwire-3d-printers

above link shows fysetc hotkey buttons product.

github link. There are klipper template config file and stls for buttons.
https://github.com/FYSETC/Hotkey/tree/main

**Comment by Surion79** [comment]
looks doable. But I need to take a look. probably won't make it in the next release, which is released in a couple days apparently.

**Comment by RadioSaebb** [comment]
Yes that would be great! I am using these too. I like you klippain very much, it helps a lot of people. I have already donated and will do it again, hope you will go further with this project. Thanks!

**Comment by mjonuschat** [comment]
Implemented in PR #813
